### DISCUSSION TITLE
MDNS not working

### DISCUSSION THREAD

**Discussion by orbitcoms** [Mair/esp32-course#200] [Course Qeustions]
I am working through the video MDNS and find the name service is not working as per the video.

Firstly, if I do not add an include for mdns in my code, the compiler does not recognise the functions.

Next, I add the include and compiile but when I run the code and ping my_esp32, the url is not found.
I can enter the url in browser 192.168.1.106 and get my hello world text ok but no response to http://my-esp32.local
![mdns issue](https://user-images.githubusercontent.com/41096656/204115339-96c8cc4e-aadc-4a4a-a45c-12f317431cd5.png)

**Comment by Mair** [comment]
hi @orbitcoms 
2 things to check.
1. is the function being called? I cant see from the screen shot.
2. are you checking it with a phone or a pc. phones don't support MDNS

**Comment by orbitcoms** [comment]
The function gets called and I am using PC to connect to the device.
It has worked in the past from my office PC but had troubles in weekend at home.
